src/pages/signin/SignInPage.js

```diff
@@ -33,149 +30,157 @@ const propTypes = {
 
         /** The primaryLogin associated with the account */
         primaryLogin: PropTypes.string,
-    }),
 
-    /** List of betas available to current user */
-    betas: PropTypes.arrayOf(PropTypes.string),
+        /** Has the user pressed the forgot password button? */
+        forgotPassword: PropTypes.bool,
+
+        /** Does this account require 2FA? */
+        requiresTwoFactorAuth: PropTypes.bool,
+    }),
 
     /** The credentials of the person signing in */
     credentials: PropTypes.shape({
         login: PropTypes.string,
         password: PropTypes.string,
         twoFactorAuthCode: PropTypes.string,
+        validateCode: PropTypes.string,
     }),
-
-    ...withLocalizePropTypes,
-
-    ...windowDimensionsPropTypes,
 };
 
 const defaultProps = {
     account: {},
-    betas: [],
     credentials: {},
 };
 
-class SignInPage extends Component {
-    componentDidMount() {
-        Performance.measureTTI();
+const SignInPage = ({account, credentials}) => {
```

Comment:
shouldn't be 
```
function SignInPage(props) { ... }
```
1. I think we wanna use `function` instead of `const`, [Slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1682443591606479), [New Doc](https://github.com/Expensify/App/pull/16882/files)

2. I think we are in discussion, there is an agreement to not destructuring it, but with another Callstack dev we think differently. [Slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1680718704040129)

Reply:
Updated to use `function` keyword. Destructuring is "officially optional": https://github.com/Expensify/eslint-config-expensify/pull/64

> Yeah, no destructuring is better. It is more readable overall

Lots of people say the opposite 🙃